BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Antenna_Performance_Metric/XFintoPUEO_Symmetric.py

```diff
@@ -0,0 +1,133 @@
+    1000.0e6, 1010.0e6, 1020.0e6, 1030.0e6, 1040.0e6, 1050.0e6, 1060.0e6,
+    1070.0e6, 1080.0e6, 1090.0e6, 1100.0e6, 1110.0e6, 1120.0e6, 1130.0e6, 
+    1140.0e6, 1150.0e6, 1160.0e6, 1170.0e6, 1180.0e6, 1190.0e6, 1200.0e6, 
+    1210.0e6, 1220.0e6, 1230.0e6, 1240.0e6, 1250.0e6, 1260.0e6, 1270.0e6, 
+    1280.0e6, 1290.0e6, 1300.0e6, 1310.0e6, 1320.0e6, 1330.0e6, 1340.0e6, 
+    1350.0e6, 1360.0e6, 1370.0e6, 1380.0e6, 1390.0e6, 1400.0e6, 1410.0e6, 
+    1420.0e6, 1430.0e6, 1440.0e6, 1450.0e6, 1460.0e6, 1470.0e6, 1480.0e6, 
+    1490.0e6, 1500.0e6
+    ]
+
+numFreq = 131 # 1500 - 200 = 1300 / 10 = 130 steps    
+
+# These are values to test with current nonPUEO files (These are the frequency values for ARA simulation software)
+'''
+freqVals = [
+    83.33e6, 100.00e6, 116.67e6, 133.33e6, 150.00e6, 166.67e6, 183.34e6,
+    200.00e6, 216.67e6, 233.34e6, 250.00e6, 266.67e6, 283.34e6, 300.00e6, 
+    316.67e6, 333.34e6, 350.00e6, 366.67e6, 383.34e6, 400.01e6, 416.67e6,
+    433.34e6, 450.01e6, 466.67e6, 483.34e6, 500.01e6, 516.68e6, 533.34e6,
+    550.01e6, 566.68e6, 583.34e6, 600.01e6, 616.68e6, 633.34e6, 650.01e6,
+    666.68e6, 683.35e6, 700.01e6, 716.68e6, 733.35e6, 750.01e6, 766.68e6,
+    783.35e6, 800.01e6, 816.68e6, 833.35e6, 850.02e6, 866.68e6,883.35e6,
+    900.02e6, 916.68e6, 933.35e6, 950.02e6, 966.68e6, 983.35e6, 1000.00e6,
+    1016.70e6, 1033.40e6, 1050.00e6, 1066.70e6
+    ]
+numFreq = 60
+'''
+
+freqList = [f'{val}' for val in freqVals] # This is in the format that pueoSim wants
+big_freq_list = freqList * 6 # This is the same thing, just 6 times :) (this is to get the gain at different angles at each frequency, we have 6 angles)
+
+# Adds arguments when running code
+parser = argparse.ArgumentParser()
+parser.add_argument("NPOP", help="How many antennas are being simulated each generation?", type=int)
+parser.add_argument("WorkingDir", help="Evolutionary_Loop directory", type=Path)
+parser.add_argument("RunName", help="Run Name directory where everything will be stored", type=str)
+parser.add_argument("gen", help="The generation the loop is on", type=int)
+parser.add_argument("indiv", help="The individual in the population", type=int)
+g = parser.parse_args()
+
+# Function to read in files
+def readFile(indiv, freqNum, col):
+    uanname = Path(f'{g.WorkingDir}/Run_Outputs/{g.RunName}/{g.gen}_{indiv}_{freqNum}.uan')
+    #uanname = Path(f'{g.WorkingDir}/{g.RunName}/uan_files/{g.gen}_uan_files/{indiv}/{g.gen}_{indiv}_{freqNum}.uan')
+    #pulling data from the uan file                                                                                         
+    data = np.genfromtxt(uanname, unpack=True, skip_header=18).tolist()
+
+    return (data[col]) #returns Gain(Theta) column THIS WILL NEED TO BE CHANGED FOR WHEN WE HAVE WORKING XF OUTPUTS (maybe make column input variable for function?)                                                                               
```

Comment:
```suggestion
    return data[col]  # returns Gain(Theta) column THIS WILL NEED TO BE CHANGED FOR WHEN WE HAVE WORKING XF OUTPUTS (maybe make column input variable for function?)                                                                               
```
The `()` are not needed.  In addition, try to stick to <=80 chars per line.  If the comment is too long, move it before the `return`, and break it into 2-3 separate lines.